tests/unit/typescriptDeclarationFile/__snapshots__/modelScalarFields.test.ts.snap

```diff
@@ -1327,6 +1382,11 @@ declare namespace $Types {
        * The documentation of this field.
        */
       description: undefined
+    
+      /**
+       * The resolver of this field
```

Comment:
Maybe we should generate resolvers for scalars. Or omit them complete. This inbetween state is a bit odd. Maybe this would make sense if the JSDoc explained what is going on.

> The resolver of this field.
> In Nexus, scalars already have default resolvers which is to select a field of the same name from the source type.

I think it would be simpler if we generate the resolver. Reasons:

1. There is currently a [Nexus bug](https://github.com/graphql-nexus/nexus/issues/827) where resolvers are needed for scalars.
2. If anyone wants to write logic against our generated types, the more symmetrical the data is the easier it will be for them I think. Generally.

Reply:
Yes, it's simpler to generate default resolvers as Nexus does. I just update the PR.